tfjs-node/scripts/deps-constants.js

```diff
@@ -83,7 +83,7 @@ if (os.platform() === 'win32') {
   destLibTensorFlowName += '.so';
   destLibTensorFlowFrameworkName += '.so';
 } else {
-  throw Exception('Unsupported platform: ' + os.platform());
+  throw new Exception('Unsupported platform: ' + os.platform());
```

Comment:
Should this be `new Error`?

Reply:
thanks for spotting this, missed a commit.